### DISCUSSION TITLE
GitHub Copilot not working for some reason

### DISCUSSION THREAD

**Discussion by markoKar** [community/community#55158] [GitHub Education] [answered]
### Select Topic Area

Bug

### Body

My Github Copilot does not seem to work, and I don't know why.

1) I am using solely my student email which should allow me the usage of Copilot for free
2) I signed out and signed in on VS Code
3) The error I'm getting:
`[ERROR] [default] [2023-05-11T10:13:06.398Z] GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as markoKar."`
`[INFO] [auth] [2023-05-11T10:14:02.707Z] Invalid copilot token: missing token: 403`

What am I supposed to do?

**Comment by davecheney** [comment, marked as answer]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by claudiojara** [reply]
where?

**Reply by davecheney** [reply]
Bottom left hand corner

**Reply by gologanalexandra21** [reply]
did this work for you? i have the same problem and after signing out and back in nothing changed

**Reply by davecheney** [reply]
@gologanalexandra21 there is no copilot subscription for your account. You can visit https://github.com/github-copilot/signup to sign up for a trial.

**Comment by gologanalexandra21** [comment]
I signed up for the Student Developer Pack few days ago but I haven't got
any confirmation yet. I suppose I have to wait for my benefits to become
available. But how long should this take? Is there another way to use
Copilot for free as a student? It seems that using GitHub with a school
email address is not enough.
Thank you!

joi, 29 feb. 2024, 22:30 Dave Cheney ***@***.***> a scris:

**Reply by davecheney** [reply]
I'll transfer this issue to GitHub Education who will be able to assist you.
